docs/index.rst

```diff
@@ -3,14 +3,24 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to open_plan's documentation!
-#####################################
+open_plan Documentation
+#######################
 
+.. image:: /logos/open_plan_logo.png
```

Comment:
This is only a query.. We do not have a logo yet right?

Reply:
We do have a logo: 
\\fs01\RL-Institut\04_Projekte\299_open_plan\05-OeA\Logo

Reply:
Added in 38ecbd1fec3213aef9a809ed47feb5bf2f2e1219, I had forgotten to commit it :)